<launch>
   <node name="navigator" pkg="mosaic_navigation" type="simple_nav.py">
      <remap from="state" to="/ground_truth/state"/>
      <remap from="reset" to="/ardrone/reset"/>
      <remap from="land" to="/ardrone/land"/>
      <remap from="control" to="/cmd_vel"/>
      <param name="publish_frequency" type="double" value="2.0" />
      <param name="min_speed" type="double" value="0.0" />
      <param name="max_speed" type="double" value="1.0" />
      <param name="speed_scale" type="double" value="0.2" />
   </node>
</launch>
